src/core/server/graph/schema/schema.graphql

```diff
@@ -2779,6 +2779,12 @@ type UserModerationScopes {
   on. When null, it means that the user is allowed to moderate on all sites.
   """
   sites: [Site!]
+
+  """
+  sites (when not null) list the sites that the user is only allowed to moderate
+  on. When null, it means that the user is allowed to moderate on all sites.
+  """
+  siteIDs: [String!]
```

Comment:
exposing these to the frontend because we have to compute these all over the place.